src/_internal/icon/src/Icon.tsx

```diff
@@ -19,11 +20,28 @@ export default defineComponent({
       type: String,
       required: true
     },
+    size: {
+      type: String as PropType<'small' | 'medium' | 'large'>,
+      default: 'medium'
+    },
+    color: {
```

Comment:
Do not add `color` prop to `BaseIcon` since it will make inner svg inherit the color of the wrapper dom. Set color by inline style in `Rate` instead.

For example `<n-base-icon style="color: red;" />` will make the inner svg red.